traces/traces.go

```diff
@@ -0,0 +1,77 @@
+package traces
```

Comment:
(Nit) Could we add a package comment, the usual:
```go
// Package traces ...
```

Reply:
Updated -- let me know if you'd like any more info here

Reply:
Looks good!